pyirf/tests/test_interpolation.py

```diff
@@ -0,0 +1,129 @@
+import pyirf.interpolation as interp
+import numpy as np
+import astropy.units as u
+import pytest
+from astropy.table import QTable
+
+def test_compare_irf_cuts():
```

Comment:
The test should then also go into `pyirf/tests/test_cuts.py` so the module of the function matches the corresponding test module.

Reply:
done